homeworks/nikolay.krasniuk_Kingstar22/6-adaptive/index.html

```diff
@@ -0,0 +1,267 @@
+                        <p class="news__learn-text">Learn More</p>
+                        <img src="img/news/arrow.svg" alt="arrow_learn">
+                    </a>
+                </article>
+                <article class="news__post">
+                    <picture class="news__post-picture">
+                        <source srcset="img/news/2_news2x.png">
+                        <img class="news__post-img" src="img/news/2_news.png" alt="post_img">
+                    </picture>
+                    <div class="news__signature">
+                        <p class="news__signature-text">Apps Design</p>
+                        <div class="news__signature-views">
+                            <img src="img/news/revision.svg" alt="revision">
+                            <p class="news__signature-views-text">2953</p>
+                        </div>
+                    </div>
+                    <h3 class="news__post-title">Your customer do need comprehensive course to learn how to use your app.</h3>
+                    <p class="news__post-description">Let’s be honest. Finance isn’t exactly pinnacle fun.Online companies introduced gamification elements to their products reach up to a 700% increase in conversion rates.</p>
+
+                    <a class="news__learn" href="#">
+                        <p class="news__learn-text">Learn More</p>
+                        <img src="img/news/arrow.svg" alt="arrow_learn">
+                    </a>
+                </article>
+                <article class="news__post">
+                    <picture class="news__post-picture">
+                        <source srcset="img/news/3_news2x.png">
+                        <img class="news__post-img" src="img/news/3_news.png" alt="post_img">
+                    </picture>
+                    <div class="news__signature">
+                        <p class="news__signature-text">Product Experience</p>
+                        <div class="news__signature-views">
+                            <img src="img/news/revision.svg" alt="revision">
+                            <p class="news__signature-views-text">7980</p>
+                        </div>
+                    </div>
+                    <h3 class="news__post-title">Scale your startup through exceptional product experience</h3>
+                    <p class="news__post-description">Our UX design agency creates solutions to address product usability conversion rates an the ever-present challenges of user-retention & engagement.</p>
+                    <a class="news__learn" href="#">
+                        <p class="news__learn-text">Learn More</p>
+                        <img class="news__learn-arrow" src="img/news/arrow.svg" alt="arrow_learn">
+                    </a>
+                </article>
+            </main>
+        </section>
+        <section class="clients">
+            <h2 class="clients__title">Our Clients</h2>
+            <main class="clients__content">
```

Comment:
This part is not adaptive on ipad. Could you make it wrap on small screens?

Reply:
done